src/dlgTriggerEditor.cpp

```diff
@@ -659,24 +659,31 @@ void dlgTriggerEditor::slot_viewErrorsAction()
 }
 
 
-void dlgTriggerEditor::setTBIconSize(int s)
+void dlgTriggerEditor::slot_setToolBarIconSize(const unsigned int s)
```

Comment:
Hey, let's not add unsigned ints here. It's not an optimisation we need and Bjarne Stroustrup's experience is that [they're error prone](https://stackoverflow.com/questions/30395205/why-are-unsigned-integers-error-prone).

Reply:
Though so will be trying to use a negative (or zero) value here - OK will fix this...